test/tests/materials/isoil/tests

```diff
@@ -232,16 +265,28 @@
     # 1 element problem to check stiffness pressure dependency
     type = RunException
     input = HYS_pressure_dependent_stiffness.i
+    verification = 'isoil_verification.md'
+    expect_err = "Pressure dependency is set to true but a0, a1 and a2 are set to 1\.0, 0\.0 and 0\.0, respectively\. Strength pressure dependency is NOT turned on\."
+
+    requirement = "The ISoilAction and ComputeISoilStress class shall accurately adjust the stress-strain curve to account for pressure dependent changes in stiffness of the soil, but not changes in strength."
+  [../]
+
+  [./pressure_dependent_strength]
+    # 1 element problem to check strength pressure dependency
+    type = RunException
+    input = HYS_pressure_dependent_strength.i
+    verification = 'isoil_verification.md'
     expect_err = "Pressure dependency is set to true but b\_exp is set to 0\.0\. Stiffness pressure dependency is NOT turned on\."
 
-    requirement = "The ComputeISoilStress class shall generate a warning if pressure dependency is set to true but b\_exp is set to 0.0."
+    requirement = "The ISoilAction and ComputeISoilStress class shall accurately adjust the stress-strain curve to account for pressure dependent changes in strength of the soil, but not changes in stiffness."
```

Comment:
Please remove, 'but not changes in stiffess'. the stiffness (at the same strain) will change as well, when the strength is increased.